src/eval.py

```diff
@@ -90,11 +102,12 @@ def evaluate(
         if isinstance(finetuned_model, BaseEstimator):
             assert isinstance(pretrained_model, Presto)
 
+        val_ds = WorldCerealLabelledDataset(self.val_df)
         dl = DataLoader(
-            WorldCerealLabelledDataset(self.val_ds),
+            val_ds,
             batch_size=8192,
-            shuffle=False,
-            num_workers=2,
+            shuffle=False,  # keep as False!
```

Comment:
We could add an assert here to make sure this is the case:
`assert isinstance(dl.sampler, torch.utils.data.SequentialSampler)`